src/sys/linux.rs

```diff
@@ -94,8 +97,8 @@ impl Awake {
         self.idle_fd = if self.options.idle {
             Some(self.manager_proxy.as_ref().unwrap().inhibit(
                 "idle",
-                "keepawake-rs",
-                "User requested",
+                self.options.consumer.as_deref().unwrap_or("keepawake-rs"),
```

Comment:
Would probably make sense to make these methods of `AwakeOptions`, so as to not duplicate this logic. Only if you are up to it as I will likely refactor `AwakeOptions` into a builder later on.